tests/test_base_interface.py

```diff
@@ -1,16 +1,14 @@
 import pytest
```

Comment:
commit naming: 
best practice is using a short descriptive commit message and if necessary elaborate further in the body part (so no line breaking from message header to message body)